ios-target/Model/TargetAnnotation.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TargetAnnotation.swift
+//  ios-target
+//
+//  Created by sol manini on 6/21/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+enum TargetType: Int {
+  case christmas = 0
+  case pokemon
+  case football
+  case travel
+  case politics
+  case art
+  case dating
+  case music
+  case movies
+  case series
+  case food
+  
+  func image() -> UIImage {
+    switch self {
+    case .christmas:
+      return UIImage(named: "christmas-pin") ?? UIImage()
```

Comment:
To avoid repetitions you could have a computed variable to return the image name only and then this function returning `UIImage(named: imageName) ?? UIImage()`